### PR TITLE
Another cleanup.

### PR DESCRIPTION
**Pull request by phrohdoh**
Remove explicit private.
Remove unnecessary delegate.
Use extension method syntax.
Fix HealthInfo.NotifyAppliedDamage’s desc.
Remove unused using directives.
Remove explicit type declaration in Manifest.

### CODE REVIEW

**Comment by Phrohdoh** [comment]
`Not found` thanks appveyor.

**Comment by RoosterDragon** [comment]
:+1:

**Comment by Phrohdoh** [comment]
I will fake-out appveyor a bit later.

File: OpenRA.Game/Traits/Health.cs [hunk 1]
```diff
@@ -122,7 +122,7 @@ public void InflictDamage(Actor self, Actor attacker, int damage, DamageWarhead
 				damage = Util.ApplyPercentageModifiers(damage, modifiers);
 			}
 
-			hp = Exts.Clamp(hp - damage, 0, MaxHP);
+			hp = hp - damage.Clamp(0, MaxHP);
```
Comment (Mailaender) [inline OpenRA.Game/Traits/Health.cs]: I think this has to be `hp = (hp - damage).Clamp(0, MaxHP);` to not change the calculation.

Comment (RoosterDragon) [inline OpenRA.Game/Traits/Health.cs]: Yeah you're right, nice spot.

Comment (Phrohdoh) [inline OpenRA.Game/Traits/Health.cs]: I wasn't quite sure about this but that makes sense, I'll fix this.

**Comment by Phrohdoh** [comment]
Updated.

**Comment by Phrohdoh** [comment]
It's a good thing we have AppVeyor.

File: OpenRA.Game/Traits/Health.cs [hunk 2]
```diff
@@ -18,10 +18,10 @@ public class HealthInfo : ITraitInfo, UsesInit<HealthInit>
 		[Desc("HitPoints")]
 		public readonly int HP = 0;
 
-		[Desc("Physical size of the unit used for damage calculations.  Impacts within this radius apply full damage")]
+		[Desc("Physical size of the unit used for damage calculations. Impacts within this radius apply full damage")]
```
Comment (penev92) [inline OpenRA.Game/Traits/Health.cs]: Tiny nit: can you add a fullstop at the end of the description?

Comment (Phrohdoh) [inline OpenRA.Game/Traits/Health.cs]: Yep.

**Comment by penev92** [comment]
The label is for Rooster's +1, and here's one from me: :+1:

**Comment by Phrohdoh** [comment]
Updated.

**Pull request merged**